atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/TimeLockMigrationEteTest.java

```diff
@@ -44,6 +44,8 @@
 import org.junit.rules.RuleChain;
 import org.junit.rules.TemporaryFolder;
 
+// It is possible this test is run only in CI when a new version is released. Check the Circle CI configuration to allow
+// list a branch name.
```

Comment:
A better way would be to have this show up in all failures of this class. That way if someone runs to this in develop/tag release, this is more discoverable.